### ISSUE TITLE
project status

### ISSUE THREAD

**Issue by dnplkndll** [Grandcruwijnen/PrintNode-Docker#1]
Did you just not get to this or a tech issue prevent the concept?

**Comment by JKetelaar** [comment]
Good question. I did get around to getting everything into a Dockerfile, but haven't had the time to sync it up to Git yet. Are you interested? If so, I'll try to find some time this weekend.

**Comment by dnplkndll** [comment]
Yes, just wondering what you use for os base since you need drivers. We have some zebra thermal printers and lasers. 
Hoping to be able to push anywhere needed with a lightweight image and printer making for the site/ name / ip.
